cypress/integration/hosts/update_status_modal.ts

```diff
@@ -36,28 +35,4 @@ describe("Update Status Modal", () => {
 
     cy.dataCy("host-status-notes").invoke("val").should("eq", "");
   });
-
-  it("Clears form values when modal is closed", () => {
```

Comment:
Is this being tested in some other way?

Reply:
Yes the previous test already tests for this https://github.com/evergreen-ci/spruce/pull/752/files#diff-0eaef7d1d21a157b987896b82ee405a1b16bf7d9a75732be6305649d51e305f7R30-R36 
All this is doing is is repeating what already happens in the previous test with a different test name